jekyll/_cci2/orb-author.md

```diff
@@ -9,279 +9,276 @@ version:
 - Cloud
 ---
 
+* TOC
+{:toc}
+
 ## Introduction
 
-Once you have configured your platform to work with CircleCI orbs, you are ready to begin authoring your own orb. CircleCI has attempted to make this process simple and easy, eliminating any potential confusion by enabling you to use inline templates and other pre-configured templates to quickly author a new orb. The sections below describe the process of authoring your own orb.
+This orb authoring guide assumes you have read the [Introduction to authoring an orb]({{site.baseurl}}/2.0/orb-intro) document and claimed your namespace. At this point you are ready to develop and orb.
 
-## Orbs Quickstart
+Whether you are writing your first orb or getting ready for production level, we recommend using our [orb development kit](#orb-development-kit) to get started. Alternatively, as orbs are packages of [reusable configuration]({{site.baseurl}}/2.0/reusing-config), they can be written [manually]({{site.baseurl}}/2.0/orb-author-validate-publish), as singular `yaml` files, and published using our [circleci orb cli]({{site.baseurl}}/2.0/local-cli/#installation).
```

Comment:
@KyleTryon this is what I mean about linking to the steps for simple single yaml orb creation. Does {{site.baseurl}}/2.0/orb-author-validate-publish already show the right steps or shall we change that doc to have the current contents of orb-author?